red-hat-data-services/odh-deployer
# Remove dspo from odh-deployer.

## Description
Removing dspo from odh deployer 

# Testing instructions
Deploy via olm install and confirm dspo is not deployed in redhat-ods-applications.

## Merge criteria:
  - https://issues.redhat.com/browse/RHODS-7944
- [] The Jira story is acked.
